util.js

```diff
@@ -40,3 +40,15 @@ exports.getAvaConfig = filepath => {
 exports.isFunctionExpression = node => {
 	return node && functionExpressions.indexOf(node.type) !== -1;
 };
+
+exports.getTestModifiers = function getTestModifiers(node) {
```

Comment:
I really don't like that way of calling functions, as you're mixing how to call functions with how/whether they are exported, even though it's easy to disconnect the two.
I see it all the time at work, and it's frustrating every time :p